generators/typescript/sdk/client-class-generator/src/GeneratedSdkClientClassImpl.ts

```diff
@@ -880,6 +880,9 @@ export class GeneratedSdkClientClassImpl implements GeneratedSdkClientClass {
         }
 
         if (this.isRoot && this.oauthAuthScheme != null && context.generateOAuthClients) {
+            const hasQuestionToken =
```

Comment:
chatted offline, it might be nicer if these null checks were divorced so you can require one and not the other

Reply:
Good call, ended up also refactoring the other relevant call-site that tied them together.